src/python-common/setup.py

```diff
@@ -1,6 +1,16 @@
+import sys
+
 from setuptools import setup, find_packages
 
 
+if sys.version_info >= (3,0):
+    mypy = ['mypy', 'pytest-mypy']
+    pytest = 'pytest >=2.1.3'
+else:
+    mypy = []
+    pytest = 'pytest >=2.1.3,<5'
```

Comment:
Just for a record.
https://docs.pytest.org/en/latest/py27-py34-deprecation.html